x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_embeddable_chart/timeseriesexplorer_embeddable_chart.js

```diff
@@ -559,6 +564,44 @@ export class TimeSeriesExplorerEmbeddableChart extends React.Component {
               })
             );
           });
+
+        // Plus query for forecast data if there is a forecastId stored in the appState.
+        if (selectedForecastId !== undefined) {
+          awaitingCount++;
+          let aggType = undefined;
+          const detector = selectedJob.analysis_config.detectors[detectorIndex];
+          const esAgg = mlFunctionToESAggregation(detector.function);
+          if (modelPlotEnabled === false && (esAgg === 'sum' || esAgg === 'count')) {
```

Comment:
we should use `ML_JOB_AGGREGATION` here for the agg types.
It could also be written as a ternary statement to avoid using `let`

Reply:
Updated in 53b573ff6d8d57acfe1ca468df74f5a8d63354a3

Reply:
Sorry, I just noticed I got this wrong. It should be using  `ES_AGGREGATION`, as that is what is returned from `mlFunctionToESAggregation`